### DISCUSSION TITLE
어제랑 같은 조건인데, 오류창이 발생하네요.

### DISCUSSION THREAD

**Discussion by aodhan77** [changdoc/naver-vaccine-macro#190] [Q&A]
매크로 오류는 아닌거 같습니다.

1.1.1 사용중이고 2개 페이지, 5.1초 새로고침중인데

애러페이지떳다고 알람이와서 보니, 새로고침하다가

한번씩 오류가 발생했습니다(네이버 페이지화면)이 뜨더라구요.

그 상태로 새로고침 다시 자동으로 들어가면서 예약을 돌리던데, 오늘따라 이 오류가 발생했다는 페이지가 자주뜨네요.

어제 하루종일 돌릴때는 없었는데..

매크로 오류라기 보다는 네이버 사람이 몰려서 그런거 같기도 해요 ㅠ

**Comment by aodhan77** [comment]
확인해보니, 그 병원 페이지에 사람이 많이 몰리는 듯한 느낌이네요 ㅠ

**Reply by changdoc** [reply]
네 예약 신청 페이지 조회도 원천 데이터는 질병관리청 으로 부터 네이버와 카카오가 중계하는 입장이라 질병 관리청 서버의 응답이 지연되면 네이버 신청 페이지에 오류 페이지가 나오는 구조로 알고 있습니다.
